apps/web/playwright/booking/checkboxQuestione2e/checkbox-and-checkbox-question.e2e.ts

```diff
@@ -0,0 +1,31 @@
+import { test } from "../../lib/fixtures";
+import { initialCommonSteps } from "../utils/bookingUtils";
+
+test.describe.configure({ mode: "parallel" });
+
+test.afterEach(({ users }) => users.deleteAll());
```

Comment:
Do we actually need to clean up the users after each test? Are there data collisions if we don't? Just want to make sure we don't waste time if not needed.